src/renderer/src/routes/pages/Wallets/index.tsx

```diff
@@ -44,14 +30,14 @@ export const WalletsPage = () => {
   const { wallets } = useWalletsSelector()
   const { accounts } = useAccountsSelector()
   const dispatch = useAppDispatch()
-  const balanceExchange = useBalancesAndExchange(accounts)
   const { modalNavigateWrapper } = useModalNavigate()
-  const { bsAggregator } = useBsAggregatorSelector()
 
   const [selectedWallet, setSelectedWallet] = useState<IWalletState | undefined>(wallets[0])
   const [isReordering, setIsReordering] = useState(false)
   const [selectedAccount, setSelectedAccount] = useState<IAccountState | undefined>(undefined)
-  const [transactions, setTransactions] = useState<TransactionResponse[]>([])
+
+  const balanceExchange = useBalancesAndExchange(accounts)
+  const balanceExchangeAccountData = useBalancesAndExchange(selectedAccount ? [selectedAccount] : accounts)
```

Comment:
Shouldn't it also consider when the wallet is selected?

```suggestion

  const selectedWalletAccounts = useMemo(() => {
    return accounts.filter(account => account.idWallet === selectedWallet?.id)
  }, [accounts, selectedWallet?.id])

  const balanceExchangeAccountData = useBalancesAndExchange(selectedAccount ? [selectedAccount] : selectedAccounts ? selectedWalletAccounts : accounts)
```